### ISSUE TITLE
Warn when passing what looks like multiple arguments to a single std::process::Command::arg(...) call

### ISSUE THREAD

**Issue by HayleyDeckers** [rust-lang/rust-clippy#10316]
### What it does

Emit a warning when an argument of the form `"-x foo"`, `"--abc bar"`, `/X baz`, or a concatenation of these is passed to [std::process::Command::arg(...)](https://doc.rust-lang.org/std/process/struct.Command.html#method.arg) and suggest to use either multiple `arg` calls or a single `args` call instead.

 In these cases the programmer likely intended to pass multiple args to the process, and probably wants to use something like [`std::process::Command::args(["-x", "foo"]`)](https://doc.rust-lang.org/std/process/struct.Command.html#method.args) instead.

### Lint Name

possible_multiple_args_in_arg

### Category

suspicious

### Advantage

- prevent unintended behavior when executing other processes with `Command`.

### Drawbacks

This lint will by design always have some false-positives as Clippy can't be sure this _isn't_ what you wanted to do.

### Example

```rust
std::process::Command::new("mkdir")
        .arg("-p foo/bar")
```

Could be written as:

```rust
std::process::Command::new("mdkir")
        .args(["-p", "foo/bar"])
```
